packages/bratus-app/src/components/ComponentNode/ComponentNode.jsx

```diff
@@ -82,38 +85,65 @@ const ComponentNode = (node) => {
       : nodeName;
   };
 
-  const truncatedNodeName = truncateNodeName(node.data.label, nodeNameLength);
+  const content = (
```

Comment:
@stocksking  Now that it's done, move this content to a separate component called <PopoverContent /> and assign this to the content props instead. We should avoid having jsx code assigned to a variable like that.